collect_app/src/androidTest/java/org/odk/collect/android/regression/FillBlankFormTest.java

```diff
@@ -611,4 +620,160 @@ public void questions_shouldHavePrefilledValue() {
+                .checkIsTextDisplayed("File: /storage/emulated/0/odk/forms/search_and_select-media/nombre2.csv is missing.")
+                .swipeToNextQuestion()
+                .clickSaveAndExit()
+
+                .startBlankForm("cascading select test")
+                .clickOnText("Texas")
+                .swipeToNextQuestion()
+                .checkIsTextDisplayed("File: /storage/emulated/0/odk/forms/select_one_external-media/itemsets.csv is missing.")
+                .swipeToNextQuestion()
+                .checkIsTextDisplayed("File: /storage/emulated/0/odk/forms/select_one_external-media/itemsets.csv is missing.")
+                .swipeToNextQuestion(4)
+                .clickSaveAndExit()
+
+                .startBlankForm("fieldlist-updates")
+                .clickGoToArrow()
+                .clickGoUpIcon()
+                .clickOnElementInHierarchy(14)
+                .clickOnText("Source15")
+                .checkIsTextDisplayed("File: /storage/emulated/0/odk/forms/fieldlist-updates_nocsv-media/fruits.csv is missing.")
+                .swipeToNextQuestion()
+                .clickSaveAndExit();
+    }
+
+    @Test
+    public void changedName_shouldNotDisappearAfterScreenRotation() {
+        //TestCase13
+        new MainMenuPage(rule)
+                .startBlankForm("All widgets")
+                .clickGoToArrow()
+                .clickJumpEndButton()
+                .clickOnId(R.id.save_name)
+                .inputText("submission")
+                .closeSoftKeyboard()
+                .rotateToLandscape(new FormEntryPage("All widgets", rule))
+                .checkIsTextDisplayed("submission")
+                .rotateToPortrait(new FormEntryPage("All widgets", rule))
+                .checkIsTextDisplayed("submission");
+    }
+
+    @Test
+    public void backwardButton_shouldNotBeClickable() {
+        //TestCase14
+        new MainMenuPage(rule)
+                .startBlankForm("All widgets")
+                .checkAreNavigationButtonsNotDisplayed()
+                .clickOptionsIcon()
+                .clickGeneralSettings()
+                .clickOnUserInterface()
+                .clickNavigation()
+                .clickUseNavigationButtons()
+                .pressBack(new GeneralSettingsPage(rule))
+                .pressBack(new FormEntryPage("All widgets", rule))
+                .checkBackNavigationButtonIsNotsDisplayed()
+                .checkNextNavigationButtonIsDisplayed()
+                .rotateToLandscape(new FormEntryPage("All widgets", rule))
+                .checkBackNavigationButtonIsNotsDisplayed()
+                .checkNextNavigationButtonIsDisplayed()
+                .rotateToPortrait(new FormEntryPage("All widgets", rule))
+                .checkBackNavigationButtonIsNotsDisplayed()
+                .checkNextNavigationButtonIsDisplayed();
+    }
+
+    @Test
+    public void groups_shouldBeVisibleInJumpScreen() {
```

Comment:
`JumpScreen` is the same like `hierarchyView`? We should use one name across all tests.